Cartfile

```diff
@@ -1,5 +1,5 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json"
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 3.1.4
```

Comment:
My understanding is that this library will respect semver, so it’s OK to float on the minor version as well as the patch version, just as we do in the map SDK.

/cc @kevinkreiser

Reply:
The podspec specifies `~> 3.0`, so let’s be consistent with that.